README.md

```diff
@@ -1,3 +1,65 @@
 # cdn-configuration
 
-:construction: under construction :construction:
+## Overview
+
+https://github.com/ampproject/cdn-configuration stores the AMP versions that are being served by AMP CDNs in `configs/versions.json`. It also hosts the deployment and patching workflows for the release on-duty engineer.
+
+These workflows replace the actions on the internal release dashboard. It does not, however, replace the error monitoring systems. Please check the internal doc for links to monitor errors and performance.
```

Comment:
I feel like this paragraph should either be removed or moved down to a "footnote for Googler" or something like that (in which case, explicitly mention that this is part of the transition process to open-source)